ui/panels-plugin/src/plugins/time-series-chart/utils/data-transform.ts

```diff
@@ -55,6 +57,14 @@ export function getLineSeries(
 ): EChartsTimeSeries {
   const lineWidth = visual.line_width ?? DEFAULT_LINE_WIDTH;
   const pointRadius = visual.point_radius ?? DEFAULT_POINT_RADIUS;
+
+  // Shows datapoint symbols when selected time range is roughly 15 minutes or less
+  let showPoints = data.length <= HIDE_DATAPOINTS_LIMIT;
```

Comment:
What are datapoint symbols?

Reply:
I think this is the little circle shown at points. It's called a "symbol" because you can use a variety of shapes. @sjcobb, correct me if I got that wrong.

Reply:
Yup that's right! [Example](https://echarts.apache.org/en/option.html#series-line.symbol) of supported symbols, I went with ['circle'](https://github.com/perses/perses/blob/v0.30.0/ui/components/src/utils/theme-gen.ts#L127) at the beginning instead of EChart's default `emptyCircle` since I ran benchmarks and `circle` is slightly faster

The big change here is now datapoints are visible when there aren't a ton of them (closer to how Grafana works now too)

![image](https://github.com/perses/perses/assets/9369625/acd67342-39f8-47df-9b70-6dfe96a0648b)